### PR TITLE
Add operation and test for it

### PR DESCRIPTION
**Pull request by bam241**
self explanatory

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,219 @@
+version: 2.1
+
+
+# Executors define environment where jobs will be run.
+executors:
+    py2:
+        docker:
+            - image: pyne/ubuntu_18.04_py2_pyne-deps
+        environment:
+            PYTHONPATH: /root/project/
```
Comment (kkiesling) [inline .circleci/config.yml:10]: I think we decided using PYTHONPATH here is bad practice right?

**Review by kkiesling — COMMENTED** [review]
I am actually legitimately confused about the purpose of this snippet going into this repo? Why are there so many CI jobs? Why are we adding just a simple addition operator? Is this meant to be used by someone or more like a template on how to add snippets in the future? So this is just an example?

**Comment by kkiesling** [comment]
Also- several CI jobs fail. Is that expected?

**Comment by bam241** [comment]
this was not intended to be merged closing :)

**Pull request closed without merging**